src/coreclr/nativeaot/Runtime/DebugHeader.cpp

```diff
@@ -163,6 +163,9 @@ extern "C" void PopulateDebugHeaders()
     MAKE_DEBUG_FIELD_ENTRY(dac_gc_heap, finalize_queue);
     MAKE_DEBUG_FIELD_ENTRY(dac_gc_heap, generation_table);
 
+    MAKE_SIZE_ENTRY(ee_alloc_context);
```

Comment:
@mikem8361 Is this going to need matching changes in SOS for native AOT? Do we need to bump the data contract version to make it work?

Reply:
Yes, this will require changes to the DCR/NativeAOT SOS and a minor version bump is needed.

Reply:
I think we'd want a major version bump because of the change to RuntimeThreadLocals below right? Code that expect gc_alloc_context to exist at RuntimeThreadLocals.m_rgbAllocContextBuffer isn't going to find that field any longer and anything that hardcoded the offset no longer has the right offset.

Reply:
I went ahead and bumped the major version